ckb/src/main/java/org/nervos/ckb/transaction/CellCollector.java

```diff
@@ -120,10 +120,11 @@ public CollectResult collectInputs(
     BigInteger changeCapacity =
         inputsCapacity.subtract(needCapacity.add(calculateTxFee(transaction, feeRate)));
     List<CellsWithAddress> cellsWithAddresses = new ArrayList<>();
+    List<String> lockHashArray = Arrays.asList(lockHashes.toArray(new String[0]));
```

Comment:
Maybe `lockHashList` instead of `lockHashArray ` will be better.

Reply:
updated: [`8bdd195` (#347)](https://github.com/nervosnetwork/ckb-sdk-java/pull/347/commits/8bdd195a0e0d7b6379e67b66552863009394e419)